### PR TITLE
HAWQ-758. Refactor insert checkinstall cases using new test framework

### PR DESCRIPTION
**Pull request by jiny2**

### CODE REVIEW

File: src/test/feature/query/test-insert.cpp [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+#include <vector>
+#include <stdio.h>
+#include <stdlib.h>
+#include <errno.h>
+#include <iostream>
+#include <string>
+
+#include "lib/command.h"
+#include "lib/common.h"
+#include "lib/data-gen.h"
```
Comment (xunzhang) [inline src/test/feature/query/test-insert.cpp]: According to this merged [pr](https://github.com/apache/incubator-hawq/pull/683), `#include "lib/data-gen.h"` -> `#include "lib/data_gen.h"`

File: src/test/feature/query/test-insert.cpp [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+#include "lib/hawq-config.h"
+#include "lib/sql-util.h"
```
Comment (xunzhang) [inline src/test/feature/query/test-insert.cpp]: `#include "lib/sql-util.h"` -> `#include "lib/sql_util.h"`

File: src/test/feature/query/test-insert.cpp [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+
+#include "gtest/gtest.h"
+
```
Comment (xunzhang) [inline src/test/feature/query/test-insert.cpp]: Please add `using hawq::test:: SQLUtility` here~

**Comment by xunzhang** [comment]
`src/test/feature/query/test-insert.cpp` -> `src/test/feature/query/test_insert.cpp`, I think we should be the same filename-style with Postgresql which uses underscores, but for makefile targets, we could use dashes. I have already modified the past code, so don't worry.

File: src/test/feature/query/test-insert.cpp (hunk 3 above)
Comment (jiny2) [inline src/test/feature/query/test-insert.cpp]: fixing it is ok.

File: src/test/feature/query/test-insert.cpp (hunk 1 above)
Comment (jiny2) [inline src/test/feature/query/test-insert.cpp]: removed

File: src/test/feature/query/test-insert.cpp (hunk 2 above)
Comment (jiny2) [inline src/test/feature/query/test-insert.cpp]: fixed

**Comment by xunzhang** [comment]
+1

**Comment by yaoj2** [comment]
＋1

**Comment by changleicn** [comment]
@jiny2 the code is in, please close the pull request.

**Pull request closed without merging**